assets/src/components/mapPage.tsx

```diff
@@ -102,21 +111,27 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
           thereIsAnActiveSearch(vehicles, searchPageState) ? (
             <SearchResults
               vehicles={vehicles}
-              selectedVehicleId={selectedVehicle?.id || null}
-              onClick={setSelectedVehicle}
+              selectedVehicleId={liveVehicle?.id || null}
+              onClick={selectVehicle}
             />
           ) : (
             <RecentSearches />
           )}
         </div>
       </div>
-
-      <div className="m-search-page__map">
+      <div className="m-map-page__map">
         <Map
           vehicles={onlyVehicles}
-          onPrimaryVehicleSelect={setSelectedVehicle}
+          onPrimaryVehicleSelect={selectVehicle}
           shapes={selectedVehicleShapes}
-        />
+        >
+          {liveVehicle && showVehicleCard ? (
```

Comment:
comment(non-blocking): I've been reading `vehicle` so many times that I thought this was a typo at first 🤪